lib/msf/core/exploit/capture.rb

```diff
@@ -250,7 +250,7 @@ def capture_sendto(payload="", dhost=nil, bcast=false, dev=nil)
       # The return value either be a PacketFu::Packet object, or nil
       def inject_reply(proto=:udp, pcap=self.capture)
         # Defaults to ~2 seconds
-        to = (datastore['TIMEOUT'] * 4) / 1000.0
+        to = ((datastore['TIMEOUT'] || 500) * 4) / 1000.0
```

Comment:
the other occurences in this file use `.to_f` too:
`to = ((datastore['TIMEOUT'] || 500).to_f * 8) / 1000.0` https://github.com/wvu-r7/metasploit-framework/blob/feature/bind_tkey/lib/msf/core/exploit/capture.rb#L314